front/app/containers/UserCustomFields/citizen/components/CustomFieldsStep/index.tsx

```diff
@@ -6,12 +6,11 @@ import { isObject } from 'lodash-es';
 import Button from 'components/UI/Button';
 import Error from 'components/UI/Error';
 import { Spinner } from '@citizenlab/cl2-component-library';
-import UserCustomFieldsForm from '../../citizen/components/UserCustomFieldsForm';
-import UserCustomFieldsFormMigrated from '../../citizen/components/UserCustomFieldsFormMigrated';
```

Comment:
I moved this 'step' component into the `UserCustomFields` container for now. Once we have all 'steps' in core, we can consider putting them in one folder with the rest of the `SignUpIn` code